apps/web/playwright.config.ts

```diff
@@ -34,10 +34,22 @@ export default defineConfig({
   },
 
   projects: [
+    // Setup project for authentication
     {
-      name: "chromium",
+      name: "setup",
+      testMatch: /.*\.setup\.ts/,
       use: { ...devices["Desktop Chrome"] },
     },
```

Comment:
but the test is name "tags.spec.ts" you should say that we only apply this setup for test inside api no ? or i don't undetrstand